packages/tailwind/src/utils/css/resolve-all-css-variables.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
why are we disabling this?

Reply:
Because of this portion here,

https://github.com/resend/react-email/blob/3ecc61b0c2cc443ebe42789a9003972e3aa51739/packages/tailwind/src/utils/css/resolve-all-css-variables.ts#L70-L73

we could also do a single get, and then check if the resulting value is not undefined